src/algebra/group/hom.lean

```diff
@@ -130,6 +130,13 @@ lemma mul_hom.coe_mk [has_mul M] [has_mul N]
 lemma monoid_hom.coe_mk [monoid M] [monoid N]
   (f : M → N) (h1 hmul) : ⇑(monoid_hom.mk f h1 hmul) = f := rfl
 
+@[simp, to_additive]
+lemma monoid_hom.to_one_hom_coe [monoid M] [monoid N] (f : M →* N) :
+  (f.to_one_hom : M → N) = f := rfl
```

Comment:
```suggestion
  (f.to_one_hom : M → N) = f := rfl

```

Reply:
This was deliberate, all the lemmas in this file are missing this whitespace (so as to group them together into their pairs / threes)